scripts/supervisord/echo.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
what is this file?

Reply:
It can be removed as it was part of our k8s-supervisor prototype project ;-)